system.c

```diff
@@ -176,6 +176,14 @@ uint8_t system_execute_line(char *line)
           else { report_ngc_parameters(); }
           break;
 
+        case 'P': // Set force sensitivity parameter. Avoids hastle of changing eeprom settings.
+          char_counter++;
+          if (!read_float(line, &char_counter, &parameter)){
```

Comment:
`line` is an array of chars. So, you're casting from a char to a float, and then from a float to a char. Why not just replace the whole thing with

```c
settings.force_sensor_level = line[++char_counter]
```

or something? The current implementation is complicated, which makes me wonder why the simple version doesn't work.